### PR TITLE
[FIX] Fix release 5.0.0 bad merge with 5.0.1

### PR DESCRIPTION
**Pull request by CHaNGeTe**
A bad merging process occurred on release 5.0.0 (AndroidX support)
This tries to fix it.

@vokhuyetOz can you also check this PR? sorry for that

### CODE REVIEW

**Comment by CHaNGeTe** [comment]
Closes https://github.com/react-native-community/react-native-video/issues/1688

**Review by jensellfors — APPROVED** [review]

**Comment by cobarx** [comment]
@CHaNGeTe I would recommend adding some notes to the README about the AndroidX change in 5.x and what the implications are in terms of minimum supported React Native version and support (or not) for pre-AndroidX setups. Having that data available will help prevent people opening issues about it.

**Comment by cobarx** [comment]
@CHaNGeTe Oops looks like you just added it to the README, my bad for not checking the change list. Is it possible to use AndroidX with older React Native versions?

**Comment by vanhungoz** [comment]
@CHaNGeTe , i will try to check it in the end of today, now i am in the office. i can't check it. sorry about that.  ( now it is the morning in Viet Nam )

**Comment by CHaNGeTe** [comment]
@cobarx I think you need react-native >=0.60 for that. And >=0.59.1 for 64bit builds.
I think the code on react-native-video (the compat library migration) is compatible with earlier versions, but if you plan to use androidX or 64bit I think the upgrade is required.
As 64bit will be a requirement from google for publishing I assume lots of people will migrate.
Maybe @vokhuyetOz and @yozman can elaborate more on this.

**Comment by karltaylor** [comment]
Can this be merged? 😇

File: examples/basic/android/gradle.properties [hunk 1]
```diff
@@ -18,3 +18,5 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
```
Comment (ashnfb) [inline examples/basic/android/gradle.properties:20]: I believe android.useDeprecatedNdk is no longer supported?

**Review by ashnfb — APPROVED** [review]

**Comment by ashnfb** [comment]
@CHaNGeTe sorry I've been offline for a bit. Looks fine to merge.

File: README.md [hunk 2]
```diff
@@ -130,7 +130,8 @@ From version >= 5.0.0, you have to apply this changes:
 dependencies {
    ...
     compile project(':react-native-video')
-    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
++    implementation "androidx.appcompat:appcompat:1.0.0"
```
Comment (ashnfb) [inline README.md]: the latest version is 1.0.2
androidx.appcompat:appcompat:1.0.2

File: examples/basic/android/gradle.properties (hunk 1 above)
Comment (CHaNGeTe) [reply examples/basic/android/gradle.properties:20]: should I remove the property then?

**Review by maxlxq — APPROVED** [review]

File: examples/basic/android/gradle.properties [hunk 3]
```diff
@@ -18,3 +18,5 @@
...
+android.useAndroidX=true
+android.enableJetifier=true
```
Comment (benjipott) [inline examples/basic/android/gradle.properties:22]: add line

**Review by benjipott — APPROVED** [review]

**Comment by benjipott** [comment]
@cobarx && @n1ru4l possible to check this PR please ?

**Comment by Dzuvan** [comment]
Sorry to bother, I was just wondering is there any ETA for this?

**Review by benoitdion — APPROVED** [review]

**Comment by CHaNGeTe** [comment]
released

**Pull request merged**